pkg/utils/k8sutil.go

```diff
@@ -282,6 +282,16 @@ func CreateK8sResources(ns, name string, spec *spec.FunctionSpec, client *kubern
 									MountPath: "/kubeless",
 								},
 							},
+							LivenessProbe: &v1.Probe{
+								InitialDelaySeconds: int32(3),
+								PeriodSeconds:       int32(1),
```

Comment:
I am considering the case of updating func once we have both old & new pods in same group then if the PeriodSeconds taking too long for a check then that makes non sense to pick up the correct pod. Is 3s or 5s better ?